docs/ReleaseNotes/Kubernetes-Operator-for-PostgreSQL-RN2.8.0.md

```diff
@@ -0,0 +1,222 @@
+
+The Operator standardizes on Patroni 4 as the only supported version and no longer honors Patroni version overrides via the `pgv2.percona.com/custom-patroni-version` annotation.
+
+However, if your Custom Resource is still at version 2.7.0, the Operator 2.8.0 will continue to run a temporary Pod to check Patroni version and use Patroni 3 if specified via the annotation for backward compatibility. But after you upgrade the Custom Resource to version 2.8.0, the `pgv2.percona.com/custom-patroni-version `annotation is ignored, and Patroni 4 is always used.
+
+This change eliminates ambiguity and ensures your cluster is deployed with a modern high-availability implementation.
+
+## Changelog
+
+### New features
+
+* [K8SPG-730](https://perconadev.atlassian.net/browse/K8SPG-730) - Added the  `status.observedGeneration` field to the Custom Resource Definition to improve observability and ensure the controller successfully reconciled the latest changes to the cluster.
+
+* [K8SPG-752](https://perconadev.atlassian.net/browse/K8SPG-752) - Allowed setting `loadBalancerClass` service type and using a custom implementation of a load balancer rather than the cloud provider default one.
+
+* [K8SPG-768](https://perconadev.atlassian.net/browse/K8SPG-768) Introduced a mechanism to prevent excessive logging caused by continuous pod annotation updates for suggested volume sizing. The Operator now skips updating the Pod annotation with the suggested volume size unless the auto-growable disk feature is explicitly configured. This significantly reduces redundant logs and unnecessary load on both the Kubernetes API and the logging pipeline.
+
+* [K8SPG-832](https://perconadev.atlassian.net/browse/K8SPG-832) - Users can now specify custom sidecar containers for the `repo-host` Pod, enabling seamless integration with external tools, storage systems, or observability agents. This enhances flexibility in backup workflows without modifying the Operator’s core logic.
+
+* [K8SPG-833](https://perconadev.atlassian.net/browse/K8SPG-833) - Added the ability to define custom environment variables across all components. This enables tighter integration with external systems, secrets, or runtime configurations.
+
+### Improvements
+
+* [K8SPG-460](https://perconadev.atlassian.net/browse/K8SPG-460) - The Operator now correctly enables and used Huge pages functionality if they are enabled on the OS level.
+* [K8SPG-570](https://perconadev.atlassian.net/browse/K8SPG-570) - The Operator now correctly respects custom user passwords defined in secrets when creating new users, and automatically adds any missing credentials.
+
+* [K8SPG-611](https://perconadev.atlassian.net/browse/K8SPG-611) - The operator now uses official Percona PostgreSQL docker images, which are compatible only with specific latest PostgreSQL versions.
```

Comment:
i think this should be a release highlight and explicitly mention supported versions